### ISSUE TITLE
Can't log in to app - JSON error?

### ISSUE THREAD

**Issue by bvernot** [Manuel-777/MTG-Arena-Tool#1242]
Hello - I just installed the app, and I can't seem to log in. I get the following error in a small box at the bottom of the app screen:

`Unexpected token < in JSON at position 0`

Here's a screenshot.
<img width="701" alt="Screen Shot 2021-04-03 at 9 42 18 PM" src="https://user-images.githubusercontent.com/9371506/113489801-88884700-94c6-11eb-94de-a409bf82d87c.png">

The error also appears in my renderer.log file - I could send an excerpt, but I'm not sure how much of it is ok to send.

Any advice? I searched for JSON in the current bugs, but didn't find anything. Thanks!

**Comment by bauris-prvt** [comment]
The current version of the app is 5.5.17 (released earlier today). Your screenshot indicates an older version.
But there was yet another change in how WOTC encodes the player data. This change came with the release of "Historic Anthology 4" on Arena (or maybe Kaldheim). The app was changed accordingly.
Maybe try release 5.5.15 or 5.5.17. See if that helps

**Comment by bvernot** [comment]
I updated to 5.5.17, but I still get the error. Incidentally, I tried to downgrade to 5.5.15 to try that out, but it still lists my version as 5.5.17. I don't think that's related to this bug, though - I'm probably just messing something up there.

I'm on macOS.

I've tried copying part of renderer.log here, replacing any personal information, to see if it's helpful:

```
[2021-04-04 09:20:43.725] [debug] --------
[2021-04-04 09:20:43.729] [debug] APP INIT
[2021-04-04 09:20:43.731] [debug] v5.5.17
[2021-04-04 09:20:43.731] [debug] Set up IPC listeners.
[2021-04-04 09:20:44.184] [debug] Db path: /Users/PERSONAL/Library/Application Support/MTG-Arena-Tool/application.db
[2021-04-04 09:20:44.190] [info]  Local DB: Loading data...
[2021-04-04 09:20:44.341] [debug] Db backup: /Users/PERSONAL/Library/Application Support/MTG-Arena-Tool/application_backup.db
[2021-04-04 09:20:44.399] [info]  Local DB: Loading data... complete.
[2021-04-04 09:20:44.400] [debug] logUri: /Users/PERSONAL/Library/Logs/Wizards Of The Coast/MTGA/Player.log
[2021-04-04 09:20:44.437] [debug] ipc prefill_auth_form
[2021-04-04 09:20:44.437] [debug] { username: 'PERSONAL', password: '', rememberMe: true }
[2021-04-04 09:20:44.656] [debug] logLoop() start /Users/PERSONAL/Library/Logs/Wizards Of The Coast/MTGA/Player.log
[2021-04-04 09:20:44.657] [debug] Initial log parse: arenaId=PERSONAL
[2021-04-04 09:20:44.657] [debug] Initial log parse: playerName=undefined
[2021-04-04 09:20:45.689] [debug] Database up to date (114), skipping download.
[2021-04-04 09:20:48.335] [debug] IPC login: {"username":"PERSONAL","password":"PERSONAL"}
[2021-04-04 09:20:48.336] [debug] ipc begin_login
[2021-04-04 09:20:48.336] [debug] httpAuth
[2021-04-04 09:20:48.336] [debug]
[2021-04-04 09:20:48.337] [debug] PERSONAL
[2021-04-04 09:20:49.151] [debug] <br />
<b>Notice</b>:  Undefined index: current in <b>/var/www/html/api/login.php</b> on line <b>38</b><br />
{"ok":true,"discord_tag":null,"token":"PERSONAL","name":null,"patreon":false,"patreon_tier":0}
[2021-04-04 09:20:49.152] [error] Unexpected token < in JSON at position 0
[2021-04-04 09:20:49.153] [debug] ipc auth
[2021-04-04 09:20:49.157] [debug] ipc login_failed
```

Maybe the issue is that I'm not on patreon? :)

**Comment by bvernot** [comment]
I also tried re-downloading database.json - didn't help.

**Comment by bvernot** [comment]
I tried logging in again today, and it worked, I could log in. I'm not sure what is different - I still have 5.5.17, not 5.5.18. I'll close in a few days if it keeps working, and update if I see the error again.

**Comment by DrTank49** [comment]
Hi,

i still have the same issue on the last version :
![image](https://user-images.githubusercontent.com/82780695/115160694-e78ab600-a099-11eb-86c7-b6b6776c65d8.png)

**Comment by riQQ** [comment]
This is an issue with new accounts and the log changes in Arena that came with the HA4 update. A fix in MTG Arena Tool is required.

**Comment by riQQ** [comment]
This should be fixed with f4e1e0310da4fa9adb932e0546301d067145e602 and v5.5.19.
